EIPS/eip-1.md

```diff
@@ -28,8 +28,8 @@ EIP stands for Ethereum Improvement Proposal. An EIP is a design document provid
  - EIP Best Practices
    - [Transfering Ownership](#transferring-eip-ownership)
  - [EIP Editors](#eip-editors)
- - [Hardfork Meta](#hardfork-meta)
-   - [Ethereum Hardforks](#ethereum-hardforks) 
+ - [Network Upgrade Meta](#network-upgrade-meta)
+   - [Ethereum Network Upgrades](#ethereum-network-upgrades) 
```

Comment:
Can we call this list network updates and still put the DAO Hardfork. It should go somewhere but I am not sure where exactly. There have been three types of "Hardforks"

Upgrades
Homestead block 1_150_000
Spurious Dragon block 2_675_000
Byzantium block 4_370_000
Constantinople block 7_280_000
Istanbul block 9_069_000
Muir Glacier block 9_200_000
Network Splits
The DAO Hardfork 1,920,000
Security Patches/Emergency
Tangerine Whistle 2,463,000
All of them need to be accounted for when creating a client. Petersburg only affects Ropten if I remember correctly.